packages/kobotoolbox/test/index.js

```diff
@@ -130,7 +202,7 @@ describe('getSubmissions', () => {
   });
 });
 
-describe('getForms', () => {
+describe.skip('getForms', () => {
```

Comment:
Yep, it's Ok to skip these for now, we'll come in and re-write them a bit later. We need to remember to remove `nock` from this adaptor. Maybe we should do it in this PR?